docs/featuretoggles/index.rst

```diff
@@ -1,3 +1,22 @@
+Open edX Django settings
+========================
+
+This is the list of Django settings defined in the ``common.py`` modules of edx-platform.
+
+LMS settings
+------------
+
+.. TODO move this to a dedicated page
```

Comment:
Should we ticket this up separately or is this something you're going to do before this PR is ready for review?

Reply:
This is something that needs to be resolved before I remove the WIP label.